civihr_employee_portal/src/Helpers/ImageHelper.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\Helpers;
+
+class ImageHelper {
+  public static function resizeForProfile($imagePath) {
+    $image = image_load($imagePath);
+
+    if (!$image) {
+      return;
+    }
+
+    image_scale_and_crop($image, 400, 400);
```

Comment:
Avoid magic numbers, I would define two constants (PROFILE_IMAGE_DEFAULT_WIDTH and PROFILE_IMAGE_DEFAULT_WIDTH & PROFILE_IMAGE_DEFAULT_HIGHT) and use them instead.